packages/form-js-viewer/assets/form-js.css

```diff
@@ -29,18 +30,24 @@
 
   --color-background: var(--color-white);
   --color-background-disabled: var(--color-grey-225-10-95);
+  --color-background-adornment: var(--color-grey-225-10-93);
```

Comment:
Was this color a requirement from Design? I know we introduced the 5-scale color schema intentionally.

Reply:
It wasn't a requirement, but differentiating it from disabled serves this purpose here:

![image](https://user-images.githubusercontent.com/17801113/204266662-8df83d51-c53a-4f99-95bb-50496eede416.png)

![image](https://user-images.githubusercontent.com/17801113/204266729-13fe3c28-a0f5-4f0c-ab62-9f6997060b3b.png)